modules/consensus.go

```diff
@@ -268,6 +272,14 @@ func (cc *ConsensusChange) AppendDiffs(diffs ConsensusChangeDiffs) {
 	cc.SiafundPoolDiffs = append(cc.SiafundPoolDiffs, diffs.SiafundPoolDiffs...)
 }
 
+// InitialHeight returns the height of the consensus before blocks are applied.
+func (cc *ConsensusChange) InitialHeight() types.BlockHeight {
+	if cc.BlockHeight == 0 {
+		return 0
+	}
+	return cc.BlockHeight - types.BlockHeight(len(cc.AppliedBlocks))
```

Comment:
doesn't this need to account for `cc.RevertedBlocks` as well?

Reply:
It doesn't need to, but it may make more sense if it did. The height is expected to be after reverting and before applying. If the subscriber doesn't use `cc.BlockHeight` directly it follows this pattern:
```go
for _, b := range cc.RevertedBlocks {
  // Revert block
}

height := cc.InitialHeight()
for _, b := range cc.AppliedBlocks {
  // Apply block
  height++
}
```

Reply:
Ah, I see. Kinda weird, but if we explain it in the docstring then it should be fine.